### DISCUSSION TITLE
Radiation vs Colonisation and Beacons balance

### DISCUSSION THREAD

**Discussion by CurePindal** [FakeFishGames/Barotrauma#5447] [Unstable feedback]
First, I have no issues with the new radiation mechanic for itself.
Sadly the radiation mechanic is somewhat conflicting/messing with the current colonisation and beacon mechanics. Some quick solutions could be possible.
- being pushed forward by radiation is contradicting to how beacons spread colonisation, which is more backwards (yes, around them, but with a delay, so its usually behind your position on the campaign map)
- beacon colonisation impact should probably impact a tad more eastwards of the beacon, because the west is radiated soon - so makes no sense to colonise there anyway
- beacons shouldnt be sidelined by removing the beacon missions. Yes, we miss them - those little wiring puzzles are fun - please keep them!
- colonisation in general should be thought a bit more "ahead of your campaign map progress". Right now its more "Oh, Im already beyond that location, no time to go back there anyway".

**Comment by Ek-Videogames** [comment]
The last point is more or less my feelings on the radiation problem.
Outpost reputation, outpost 'evolution' to different types, and new outpost creation - mostly are now happening at a point that you often never go back to those locations for it to matter in the grand scheme of things.

**Comment by italianpsychoIT** [comment]
Maybe some radioprotector medicines can improve the radiation mechanic.
Radioprotectors can buff the radiation resistance for a while. 
The developers can add an additional layers of metals for covering of the submarine. (Submarine's body update)
metal protection increases with an increase of atomic number,  Iron<Cobalt, Cobalt<Nickel.
This should facilitate the navigation in hazardous areas.
I think something like that is easier to add than to change mechanics

**Comment by Cogs16** [comment]
I have a problem with the radiation death wall because it would be an existential threat to europa lore wise.

**Comment by ViktorShield** [comment]
Radiation itself is a horrible idea(not just lorewise). The game has to make you WANT to move forward, not to FORCE you to. 
![image](https://user-images.githubusercontent.com/84222807/119598036-85a23680-be25-11eb-87cf-4d2038f88af7.png)
There are 2 bad things about this concept. First being screwed endgame, which just teleports you back like in No Man's Sky. So you'll be trapped in no-mission zone without proper equipment and money to buy some. The second bad thing is that there are subs(mine, for example), which are designed to be extremely efficient in later game, but completely useless in early game. Its often done by not adding some necessary equipment on the submarine, like plasma cutters or toolbelts. There'll be no money to buy any. It's especially hard in case you play with ITA conversion mod, which also makes uranium rods super expensive.

**Reply by CurePindal** [reply]
> 
> 
> Radiation itself is a horrible idea(not just lorewise). The game has to make you WANT to move forward, not to FORCE you to.

I have to disagree. Its totally valid to either apply push or pull mechanics.

Regardless, I wrote this topic not to complain about the radiation mechanic (or the campaign) but to make it better.
